.github/actions/go-fmt/entrypoint.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh -l
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+
+GOPATH="$(mktemp -d)"
+SRCDIR="$GOPATH/src/github.com/apache"
+mkdir -p "$SRCDIR"
+ln -s "$PWD" "$SRCDIR/trafficcontrol"
+cd "$SRCDIR/trafficcontrol"
+
+/usr/local/go/bin/go fmt ./...
+DIFF_FILE="$(mktemp)"
+git --no-pager diff >"$DIFF_FILE"
+
+if [ -s "$DIFF_FILE" ]; then
+	./misc/parse_diffs.py <"$DIFF_FILE";
+	rm "$DIFF_FILE";
+	exit 1;
+fi
+echo "No diff found, exiting" >&2
+exit 0;
```

Comment:
`exit 0` is not necessary, 0 is the default exit code

Reply:
right. That if statement logic was reversed in an earlier revision.